examples/bri-3/src/bri/communication/messagingClients/natsMessagingClient.ts

```diff
@@ -0,0 +1,61 @@
+import { Injectable, OnModuleDestroy, OnModuleInit } from '@nestjs/common';
+import { connect, NatsConnection, StringCodec } from 'nats';
+import { LoggingService } from '../../../shared/logging/logging.service';
+import { IMessagingClient } from './messagingClient.interface';
+
+@Injectable()
+export class NatsMessagingClient
+  implements IMessagingClient, OnModuleInit, OnModuleDestroy
+{
+  private nc: NatsConnection;
```

Comment:
hate to leave nits, but if possible can we change `nc` name to something more descriptive?
same for `log`, can it be `logger` or `loggingService`, since it is not log

Reply:
Starting a comment with _hate to leave nits_ does not justify leaving nits.

Reply:
Fixed

Reply:
somebody reading this from outside might get wrong impression that you dont like my comments...
sorry i made you type 3 more letters, i know its hard :D